### PR TITLE
Fixed steps for postgresql example

### PR DESCRIPTION
**Pull request by shahpratikr**
## Change Overview

- fixed kanctl installation link
- updated responses for actionset

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: examples/postgresql/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ $ helm repo add bitnami https://charts.bitnami.com/bitnami
 $ helm repo update
 
 $ kubectl create ns postgres-test
-$ helm install my-release --namespace postgres-test bitnami/postgresql
+$ helm install my-release --namespace postgres-test bitnami/postgresql --version 11.9.13
```
Comment (pavannd1) [inline examples/postgresql/README.md:26]: Shouldn't we support the default version helm picks?

File: examples/postgresql/README.md [hunk 2]
```diff
@@ -141,8 +141,8 @@ $ kanctl create actionset --action backup --namespace kanister --blueprint postg
 actionset backup-llfb8 created
 
 $ kubectl --namespace kanister get actionsets.cr.kanister.io
-NAME           AGE
-backup-glptq   38s
+NAME           PROGRESS   LAST TRANSITION TIME   STATE
+backup-glptq   10.00      2022-12-08T18:14:09Z   complete
```
Comment (pavannd1) [inline examples/postgresql/README.md:145]: The `PROGRESS` here should be marked as 100.00 instead of 10.00. I think @akankshakumari393 reported this issue as well. Could you please change this to `100.00`?

File: examples/postgresql/README.md [hunk 3]
```diff
@@ -197,12 +197,22 @@ $ kanctl --namespace kanister create actionset --action restore --from backup-gl
 actionset restore-backup-glptq-6jzt4 created
 
 ## Check status
-$ kubectl --namespace kanister describe actionset restore-backup-glptq-6jzt4
+$ kubectl --namespace kanister get actionset restore-backup-glptq-6jzt4
+NAME                         PROGRESS   LAST TRANSITION TIME   STATE
+restore-backup-glptq-6jzt4   10.00      2022-12-08T18:16:50Z   complete
```
Comment (pavannd1) [inline examples/postgresql/README.md:202]: Same comment for the progress

File: examples/postgresql/README.md [hunk 4]
```diff
@@ -235,7 +245,9 @@ $ kanctl --namespace kanister create actionset --action delete --from backup-glp
 actionset delete-backup-glptq-cq6bw created
 
 # View the status of the ActionSet
-$ kubectl --namespace kanister describe actionset delete-backup-glptq-cq6bw
+$ kubectl --namespace kanister get actionset delete-backup-glptq-cq6bw
+NAME                         PROGRESS   LAST TRANSITION TIME   STATE
+delete-backup-glptq-cq6bw    10.00      2022-12-08T18:16:50Z   complete
```
Comment (pavannd1) [inline examples/postgresql/README.md:250]: Same comment for the progress

File: examples/postgresql/README.md [hunk 5]
```diff
@@ -264,13 +276,16 @@ $ helm delete my-release -n postgres-test
 ```
 
 ### Delete CRs
-Remove Blueprint and Profile CR
+Remove Blueprint, Profile CR and Actionset
```
Comment (pavannd1) [inline examples/postgresql/README.md:279]:
```suggestion
Remove Blueprint, Profile CR and ActionSet
```

File: examples/postgresql/postgres-blueprint.yaml [hunk 6]
```diff
@@ -70,7 +70,7 @@ actions:
       name: deleteDump
       args:
         image: ghcr.io/kanisterio/postgres-kanister-tools:0.84.0
-        namespace: "{{ .Namespace.Name }}"
+        namespace: "{{ .StatefulSet.Namespace }}"
```
Comment (pavannd1) [inline examples/postgresql/postgres-blueprint.yaml:73]: Why is this required? Note that downstream [adopters](https://github.com/kanisterio/kanister/blob/master/ADOPTERS.md) of Kanister use Namespace as the subject when executing `delete` action and this change will break them. I would recommend reverting this

File: examples/postgresql/v10.16.2/postgres-blueprint.yaml [hunk 7]
```diff
@@ -70,7 +70,7 @@ actions:
       name: deleteDump
       args:
         image: ghcr.io/kanisterio/postgres-kanister-tools:0.84.0
-        namespace: "{{ .Namespace.Name }}"
+        namespace: "{{ .StatefulSet.Namespace }}"
```
Comment (pavannd1) [inline examples/postgresql/v10.16.2/postgres-blueprint.yaml:73]: Same as above

Also, I believe this blueprint is no longer relevant. We can remove the v10.16.2 and v8.6.4 folders completely. @viveksinghggits @PrasadG193 WDYT?

File: examples/postgresql/README.md [hunk 8]
```diff
@@ -13,7 +13,7 @@ Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment
 - Kubernetes 1.10+
```
Comment (pavannd1) [inline examples/postgresql/README.md:13]:
```suggestion
- Kubernetes 1.20+
```
Let's bump this up since we don't really support older versions

File: examples/postgresql/postgres-blueprint.yaml (hunk 6 above)
Comment (shahpratikr) [reply examples/postgresql/postgres-blueprint.yaml:73]: I was getting nil pointer reference for `Namespace.Name`. if downstream projects are using `Namespace.Name`, it should be consistent across phases right? we only have `Namespace.Name` for delete phase here but other 2 phases have `StatefulSet.Namespace`

File: examples/postgresql/README.md (hunk 1 above)
Comment (shahpratikr) [reply examples/postgresql/README.md:26]: fixed. updated dockerfile

File: examples/postgresql/README.md (hunk 2 above)
Comment (shahpratikr) [reply examples/postgresql/README.md]: changed

File: examples/postgresql/README.md (hunk 3 above)
Comment (shahpratikr) [reply examples/postgresql/README.md]: changed

File: examples/postgresql/README.md (hunk 4 above)
Comment (shahpratikr) [reply examples/postgresql/README.md]: changed

File: examples/postgresql/README.md (hunk 5 above)
Comment (shahpratikr) [reply examples/postgresql/README.md]: fixed

File: examples/postgresql/postgres-blueprint.yaml (hunk 6 above)
Comment (shahpratikr) [reply examples/postgresql/postgres-blueprint.yaml]: reverted the change

File: examples/postgresql/v10.16.2/postgres-blueprint.yaml (hunk 7 above)
Comment (shahpratikr) [reply examples/postgresql/v10.16.2/postgres-blueprint.yaml]: reverted the change

File: examples/postgresql/README.md (hunk 8 above)
Comment (shahpratikr) [reply examples/postgresql/README.md]: fixed

File: examples/postgresql/v10.16.2/postgres-blueprint.yaml (hunk 7 above)
Comment (viveksinghggits) [reply examples/postgresql/v10.16.2/postgres-blueprint.yaml]: sorry, maybe I am missing some context. Why this blueprint not relevant?

Comment (pavannd1) [reply examples/postgresql/v10.16.2/postgres-blueprint.yaml]: @viveksinghggits are there any specific uses for this blueprint? Do we need to support this particular version of postgres?

File: docker/postgres-tools-9.6/Dockerfile [hunk 9]
```diff
@@ -1,6 +1,4 @@
-FROM postgres:14-bullseye
-
-LABEL maintainer="vkamra@kasten.io"
+FROM postgres:15-bullseye
```
Comment (pavannd1) [inline docker/postgres-tools-9.6/Dockerfile:1]: As a follow-up, we should remove this Dockerfile and stop building this image through goreleaser. I don't think we use this anymore

**Review by pavannd1 — APPROVED** [review]

File: examples/postgresql/v10.16.2/postgres-blueprint.yaml (hunk 7 above)
Comment (viveksinghggits) [reply examples/postgresql/v10.16.2/postgres-blueprint.yaml]: I don't think we have to support that specific version. But I think it's ok to keep the stale versions of blueprint just for future reference. if someone wants to refer to specific postgres' versions blueprint.
I think this is the reason we decided to have these dirs with older blueprints right?
I am also ok if decide to remove them.
